ERCS/erc-dapp-security.md

```diff
@@ -0,0 +1,111 @@
+---
```

Comment:
```suggestion
---
eip: 7817
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename